tonic/transforms.py

```diff
@@ -99,19 +90,16 @@ class DropEvent:
                                  between 0 and drop_probability.
     """
 
-    def __init__(
-        self, drop_probability: float = 0.5, random_drop_probability: bool = False
-    ):
-        self.drop_probability = drop_probability
-        self.random_drop_probability = random_drop_probability
+    drop_probability: float = 0.5
+    random_drop_probability: bool = False
 
-    def __call__(self, events, sensor_size, ordering, images=None, multi_image=None):
-        events = functional.drop_event_numpy(
+    def __call__(self, events):
+        return functional.drop_event_numpy(
             events, self.drop_probability, self.random_drop_probability
         )
-        return events, images, sensor_size
 
 
+@dataclass
```

Comment:
This one cannot be hashed/frozen unless you make coordinates an immutable Tuple.